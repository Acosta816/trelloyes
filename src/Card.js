import React from 'react';
import './Card.css';




class Card extends React.Component {

    render(){

        return(
            <div className='Card'>
                <h3>{this.props.titleProp}</h3>
                <p>{this.props.contentProp}</p>
            </div>
        )
    }
}


export default Card;