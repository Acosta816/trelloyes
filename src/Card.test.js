import React, {Component} from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';



describe('Test Suite for Card.js', () => {

    it('Smoke: Card.js renders without crashing', ()=> {
        const div = document.createElement('div');
        ReactDOM.render(<Card />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('Snapshot#1: Card renders as expected', () => {
        const cardJson = renderer.create(<Card />).toJSON();
        expect(cardJson).toMatchSnapshot();
    });

});