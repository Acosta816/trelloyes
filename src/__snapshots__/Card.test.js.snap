// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Suite for Card.js Snapshot#1: Card renders as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
